distributed/tests/test_scheduler.py

```diff
@@ -132,115 +132,104 @@ async def test_decide_worker_with_restrictions(client, s, a, b, c):
 
 
 @pytest.mark.parametrize("ndeps", [0, 1, 4])
-@pytest.mark.parametrize(
-    "nthreads",
-    [
-        [("127.0.0.1", 1)] * 5,
-        [("127.0.0.1", 3), ("127.0.0.1", 2), ("127.0.0.1", 1)],
-    ],
+@gen_cluster(
```

Comment:
Remove useless single-worker use case.
The rest of the diff of this test is just a de-indentation.

Reply:
> [("127.0.0.1", 1)] * 5

is a case with five workers, each worker having one thread. Why is this useless?

Reply:
I misread. Reverting